src/storage/scripting.cc

```diff
@@ -127,11 +127,6 @@ void LoadFuncs(lua_State *lua, bool read_only) {
   lua_pushcfunction(lua, RedisStatusReplyCommand);
   lua_settable(lua, -3);
 
-  /* redis.read_only */
-  lua_pushstring(lua, "read_only");
-  lua_pushboolean(lua, read_only);
-  lua_settable(lua, -3);
```

Comment:
It's used for users to check if the script is running on a read-only lua vm (we have one global lua vm and several per-worker lua vm, and the latter can only run read-only scripts).

Reply:
Yes, I noticed this and used the `read_only` parameter in the function to get a read-only lua vm.
```
auto lua = read_only ? conn->Owner()->Lua() : srv->Lua();
```
For checks during script execution, I store flags in `ScriptRunCtx`, which supports more flags, not just `read-only`.
I'll go into more detail tonight and you can start reviewing it tonight or tomorrow.